src/lib/common.ts

```diff
@@ -39,3 +46,42 @@ export function firmwareUpdateOutgoingMessage<
   }
   return { result };
 }
+
+export async function setRawConfigParameterValue(
+  message:
+    | IncomingCommandNodeSetRawConfigParameterValue
+    | IncomingCommandEndpointSetRawConfigParameterValue,
+  nodeOrEndpoint: ZWaveNode | Endpoint,
+): Promise<{ result?: SupervisionResult }> {
+  if (
+    message.valueSize !== undefined
+      ? message.valueFormat === undefined
+      : message.valueFormat !== undefined
```

Comment:
Interesting construct. I'd probably use `(... && ...) || (... && ...)` to be more obvious what this does